api/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -1285,6 +1285,12 @@ type GlobalConfig struct {
 	// Configure the secret backend feature https://docs.datadoghq.com/agent/guide/secrets-management
 	// See also: https://github.com/DataDog/datadog-operator/blob/main/docs/secret_management.md
 	SecretBackend *SecretBackendConfig `json:"secretBackend,omitempty"`
+
+	// Configure whether the process Agent or core Agent collects process and/or container information (Linux only).
```

Comment:
Could we also include that the Process Agent won't run if true?

Reply:
Other checks can run on the process agent and will still run in those cases. Updated the description to reflect that.

Reply:
Thanks! One more nit, I wonder if we should capitalize "core|process Agent"? It seems we do elsewhere in our docs cc @estherk15

Reply:
@celenechang I just did a search and we capitalize Process Agent, but don't capitalize core Agent. 🤷‍♀️

Reply:
I updated it to capitalize the "Process Agent" and left "core Agent" as it is.

Reply:
Thanks @daniel-taf . I'll approve, but no promises I won't change "core" later..